security/tor/src/opnsense/service/templates/OPNsense/Tor/torrc

```diff
@@ -171,13 +188,13 @@ RelayBandwidthBurst {{ OPNsense.tor.relay.bandwithburst }} KBytes
 
 #ContactInfo 0xFFFFFFFF Random Person <nobody AT example dot com>
 
-{% if helpers.exists('OPNsense.tor.relay.directory_port') and OPNsense.tor.relay.directory_port != '' %}
-DirPort {{ OPNsense.tor.relay.directory_port }}
-{% endif %}
+DirPort {% if helpers.exists('OPNsense.tor.relay.host') and OPNsense.tor.relay.host != '' %}{{ OPNsense.tor.relay.host }}:{% endif%}{{ OPNsense.tor.relay.directory_port|default('9030') }}
```

Comment:
This should be still disabled (not set) by default. Restore the original check and add the host to the original DirPort line